r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -1965,3 +1973,421 @@ test_that("lubridate's fast_strptime", {
       collect()
   )
 })
+
+test_that("round/floor/ceiling on datetime (to nearest second)", {
+
+  skip_on_os("windows") # https://issues.apache.org/jira/browse/ARROW-13168
```

Comment:
ARROW-13168 was resolved for R (but not Python) on Windows. So this and the other below could probably be removed.